django_cloud_deploy/config.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A module about configuration files generation."""
+
+import os
+from typing import Dict, List, Optional
+import yaml
+
+
+class Configuration(object):
+    """A class to encapsulate a single configuration."""
+
+    _HEADER = '# Generated file, do not edit'
+
+    def __init__(self, django_directory_path: str):
+        if not os.path.isdir(django_directory_path):
```

Comment:
Could you document the semantics? That is, it checks if the file exists and creates it if it doesn't.

Reply:
Done.